geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -705,3 +717,18 @@ $(WMTSCAPABILITIES_FILE): tilegeneration/config.yaml
 	$(PRERULE_CMD)
 	mkdir --parent $(dir $@)
 	generate_controller --capabilities
+
+# Secrets
+
+secrets.bz2.gpg:
+	tar -jcf secrets.bz2 $^
```

Comment:
Would be clearer if you called it .tar.bz2.gpg

Reply:
Done :-)